horace_core/sqw/PixelData/@PixelData/recalc_pix_range.m

```diff
@@ -15,17 +15,22 @@
 obj.set_range(obj.EMPTY_RANGE_);
 obj.reset_changed_coord_range('coordinates');
 if obj.has_more
+    ic = 1;
+    dev = 10;
```

Comment:
Unclear variable names, what are these defining?

Reply:
first is local counter, second, indeed, could be longer, though also local